config/application.rb

```diff
@@ -12,11 +12,40 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
+class JSONAPIError < Committee::ValidationError
+  def error_body
+    {
+      errors: [
+        { status: id, detail: message }
+      ]
+    }
+  end
+
+  def render
+    [
+      status,
+      { 'Content-Type' => 'application/vnd.api+json' },
+      [JSON.generate(error_body)]
+    ]
+  end
+end
+
```

Comment:
If this changes all the error messages, do we have dead code anywhere (where the code supplied the error messages)

Reply:
Yes! From the PR desc:
> This PR could go farther by ... removing some error-checking/validation code paths in the controllers that may now effectively be dead by virtue of the work Committee is doing for us

I'd be happy to include that in this PR! I'll push that up in a bit.